oracle/files/rc.local

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+clear_user(){
+	for user in oracle; do
+		userdel --force --remove "$user" || passwd -l "$user" || true
+		rm -rf "/home/$user/"
+		rm -f "/etc/sudoers.d/oracle"
+	done
+}
+
+clean_up(){
+  rm -rf /etc/rc.local
+  rm -rf /etc/rc.d/rc.local
+
+  systemctl disable rc-local
+  systemctl stop rc-local
+	rm -rf /etc/systemd/system/rc-local.service
+	rm -rf /etc/subgid.lock
```

Comment:
Please use the same indents in whole file, like on line 4 for example.

Reply:
This one comes from the original **centos/files/rc.local**. Fixed.